onnxruntime/core/graph/model_editor_api_types.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/common/inlined_containers.h"
```

Comment:
>n/inlined_containers.h" [](http://example.com/codeflow?start=20&length=23)

nit: _fwd.h header could be a better choice since inlined maps are not used here. 

BTW, they are usually faster and do not allocate end() node even if not used.